packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1027,6 +1028,24 @@ class TextInputConnection {
     );
   }
 
+  /// Sends the coordinates of caret rect. This is only used on macOS for
+  /// positioning the accent selection menu
```

Comment:
nit: `s/only used/used/` just in case we do start using it in another embedder at some point.

Reply:
Sent a fix for the othe case in https://github.com/flutter/flutter/pull/77951